CrewChiefV4/PluginInstaller.cs

```diff
@@ -273,14 +285,26 @@ public void InstallOrUpdatePlugins(GameDefinition gameDefinition)
                             if (plugins.ContainsKey(rf2PluginFileName))
                             {
                                 //the whitespace is intended, this is how the game writes it.
-                                plugins[rf2PluginFileName][" Enabled"] = 1;
+                                if(plugins[rf2PluginFileName][" Enabled"] == 0)
+                                {
+                                    if(presentEnableMessagebox())
+                                    {
+                                        plugins[rf2PluginFileName][" Enabled"] = 1;
+                                        json = JsonConvert.SerializeObject(plugins, Formatting.Indented);
+                                        File.WriteAllText(configPath, json);
+                                    }
+                                }
                             }
                             else
                             {
-                                plugins.Add(rf2PluginFileName, new Dictionary<string, int>() { { " Enabled", 1 } });
+                                if (presentEnableMessagebox())
+                                {
+                                    plugins.Add(rf2PluginFileName, new Dictionary<string, int>() { { " Enabled", 1 } });
+                                    json = JsonConvert.SerializeObject(plugins, Formatting.Indented);
+                                    File.WriteAllText(configPath, json);
```

Comment:
Nice, this is better :)  let's write to console that we updated .json/.ini at ...  Basically my biggest concern is us changing user files without them knowing, I also still think backup is good idea still (for config files), but we don't have to agree on everything ;)